src/configs/attestation.c4m

```diff
@@ -54,12 +56,12 @@ func install_cosign() {
   return false
 }
 
-func load_attestation_binary() {
+func load_attestation_binary(download_if_not_present: bool) {
```

Comment:
The `: bool` there should be redundant; I'd expect it to be inferred from the body since the body is unambiguous, but let me know if it does break (it certainly is fine to be explicit about the type here, I don't
think this needs to change).